### PR TITLE
ci: use script to build and install chaos mesh

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
Signed-off-by: xiang <xiang13225080@163.com>

### What problem does this PR solve?

use `hack/local-up-chaos-mesh.sh` script to build and install chaos mesh in the integration test

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: .github/workflows/integration_test.yml [hunk 1]
```diff
@@ -24,7 +24,8 @@ jobs:
 
       - name: Install Chaos Mesh
         run: |
-          bash install.sh --local kind --crd ./manifests/crd.yaml
+          export CLUSTER="chart-testing"
```
Comment (WangXiangUSTC) [inline .github/workflows/integration_test.yml:27]: the default cluster name of the k8s  created by `helm/kind-action`

**Review by YangKeao — DISMISSED** [review]
LGTM

File: .github/workflows/integration_test.yml (hunk 1 above)
Comment (cwen0) [reply .github/workflows/integration_test.yml:27]: I think we also use this CI to test `install.sh`. So if we merge this pr, `install.sh` won't be tested.

Comment (WangXiangUSTC) [reply .github/workflows/integration_test.yml:27]: so we can add another ci to use the `hack/local-up-chaos-mesh.sh`? I think it's ok

**Comment by WangXiangUSTC** [comment]
@cwen0 PTAL

**Review by fewdan — APPROVED** [review]
LGTM

**Review by YangKeao — APPROVED** [review]
LGTM

**Pull request merged**